README.md

```diff
@@ -64,40 +64,68 @@ Installation Requirements:
+A production grade data services deployment consists of three machines:
+* Main machine that runs the flask app that sends the requests to the ES machines.
+* Regulome search ES
+* ENCODED region-search ES
 
-1. Install the following packages:
+1. Deploy the machines. Make sure you have activated the virtual environment created above.:
     ```
-    $ sudo apt-get update
-    $ sudo apt-get install python3-psycopg2 redis-server apache2-utils nginx
+    $python deploy/deploy.py -b <your-branch-here> -n test-data-service  --image-id ami-0787b9e5747b9e78c  --profile-name regulome --instance-type r5.2xlarge --volume-size 500
+    python deploy/deploy.py -b <your-branch-here> -n test-data-service-regulome  --image-id ami-0787b9e5747b9e78c  --profile-name regulome --instance-type r5.2xlarge --volume-size 500
+    python deploy/deploy.py -b DSERVE-77-aws-deploy -n test-data-service-encoded  --image-id ami-0787b9e5747b9e78c  --profile-name regulome --instance-type r5.2xlarge --volume-size 500
     ```
 
-2. Create a password for accessing the indexer:
+2. On each machine create a password for accessing the indexer:
     ```
     $ sudo mkdir -p /etc/apache2
-    $ sudo htpasswd -c /etc/apache2/.htpasswd admin   # replace 'admin' by any username
+    $ sudo htpasswd -c /etc/apache2/.htpasswd <your-user-name>
     ```
 
-3. Edit and copy the services for systemctl:
+   You will use this login/password to access the flower dashboard on the machines. The dashboard is accessible in `http://<public-IP>/indexer`. This is accessible to the internet, so be prudent in choosing the login/password (admin is a bad username, it is quite easy to guess).
+
+3. On the main machine add the IP addresses of the ES machines into `/home/ubuntu/genomic-data-service/config/development.cfg`. Set the value of `REGULOME_ES_HOSTS` to the private IP address of the regulome data service machine, and the value of `REGION_SEARCH_ES_HOSTS` to the private IP address of the region search data service machine.
+
+4. Create a security group allowing the main application the access to ES hosts on port 9201. This security group consists of one inbound rule with IP version `IPv4`, Type `Custom TCP`, Protocol `TCP`, Port Rance `9201` with the source <private IP of the main machine>/32. Both regulome and region-search machines need to be in this security group.
+
+5. Start each service and verify they are running correctly:
+    On the main machine:
     ```
-    $ sudo ln -s /home/ubuntu/genomic-data-service/deploy/flower.service /etc/systemd/system/flower.service
-    $ sudo ln -s /home/ubuntu/genomic-data-service/deploy/celery.service /etc/systemd/system/celery.service
-    $ sudo ln -s /home/ubuntu/genomic-data-service/deploy/genomic.servie /etc/systemd/system/genomic.service
     $ sudo systemctl daemon-reload
+    $ sudo service celery start
+    $ sudo service flower start
+    $ sudo service genomic start
+    $ chmod 777 /home/ubuntu/genomic-data-service/genomic.sock
```

Comment:
Got an error:
`chmod: cannot access '/home/ubuntu/genomic-data-service/genomic.sock': No such file or directory`

![image](https://user-images.githubusercontent.com/44071821/153910040-86001424-299f-4311-b1ba-4b91a9aa8926.png)

Reply:
Good catch, this is because we should be using `production.cfg` instead of `development.cfg` for the gunicorn. I'll fix the documentation.